content/en/blog/2023/synthetic-testing/index.md

```diff
@@ -0,0 +1,100 @@
+---
+title: Portable Synthetic HTTP Testing with OpenTelemetry
+linkTitle: Synthetic HTTP testing with OTel
+date: 2023-10-05
+author: '[Carter Socha](https://github.com/cartersocha) (Lightstep)'
+cSpell:ignore: httpcheck
+---
+
+Synthetic tests are an essential part of the Observability toolkit. They can be
+used to measure application SLAs, monitor endpoints in various geographies,
+navigate a web page like a user, or identify post deployment errors before your
+customers encounter them.
+
+This blog is going to focus on HTTP based synthetic availability testing. Many vendors offer
+various ping test options with generous free tiers but monitoring endpoints at
+enterprise scale can quickly spiral into 1000s of ping tests that can’t be
+easily transferred when migrating Observability vendors. OpenTelemetry (OTel) is
+committed to the portability of Observability data to help users maximize their
+visibility into their services and get the most out of their Observability
+investments without the need to continually recreate the same capabilities.
+
+The community is excited to continually improve the project's capabilities and
+OTel has now expanded our coverage to synthetics. As of the [v0.63.0
```

Comment:
Since 0.63 has been a while the "has now expanded" feels a little bit off, can you change that sentence a little bit?

Reply:
Agreed. Reworded it slightly